src/common/helper/env.helper.ts

```diff
@@ -0,0 +1,15 @@
+import { existsSync } from 'fs';
+import { resolve } from 'path';
+
+export function getEnvPath(val: string): string {
+  const env: string | undefined = process.env.NODE_ENV;
+  const fallback: string = resolve(`${val}/.env`);
+  const filename: string = env ? `${env}.env` : 'development.env';
```

Comment:
I need you need to fix this? Because now environment file starts with .env.{environment} name?